filebeat/docs/stdin-input.asciidoc

```diff
@@ -0,0 +1,24 @@
+:type: stdin
+
+[[stdin-input]]
+=== Stdin input
+
+++++
+<titleabbrev>Stdin</titleabbrev>
+++++
+
+Use the `stdin` input to read events from standard in.
+
+Example configuration:
+
+["source","yaml",subs="attributes"]
+----
+{beatname_lc}.inputs:
+- type: stdin
+----
+
+// REVIEWERS: Are there any stdin-specific config options, or are all the config
+// options inherited from libbeat?
```

Comment:
right now it supports all harvester settings (mostly the same as are available for `log` input type). Some settings just don't make sense for stdin.

Reply:
@urso Can you look through the document I created and tell me which options make sense for stdin? I put a question mark for the options *I think* are applicable. https://docs.google.com/spreadsheets/d/1ixRNLA22LiBAgXGZzDBPxz-7kflaSn-JOQkSIcU-d78/edit#gid=0